hr_period/__openerp__.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding:utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Savoir-faire Linux. All Rights Reserved.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'HR Payroll Period',
+    'version': '1.0',
+    'license': 'AGPL-3',
+    'category': 'Generic Modules/Human Resources',
+    'description': """
```

Comment:
Remove description, since it will be filled with the README.
You could consider having a one line `summary` though.